README.md

```diff
@@ -35,35 +41,67 @@ See here for [installation instructions for Docker](https://docs.docker.com/get-
 
 Docker Compose is typically included with Docker Desktop installations, but if you need to install it separately, see [the Docker Compose installation instructions](https://docs.docker.com/compose/install/).
 
-### Ollama
+### (Optional) Ollama for local LLM models
 
-To run models locally, you'll need to have [Ollama](https://ollama.com/) installed on your machine.
+If you want to run LLM models locally, you'll need to have [Ollama](https://ollama.com/) installed on your machine.
 Once you have Ollama installed, you want to run the Ollama server, which will allow the application to access the model.
-If Ollama server is not running as a system service already, you have to start it manually:
+Make sure the Ollama server is listening to all addresses so you can use our Docker images.
+If the Ollama server is running as a system service already, stop it first (in Ubuntu Linux, you'll need to run `sudo systemctl stop ollama.service`).
+Then, start it manually:
 
 ```bash
-ollama serve
+OLLAMA_HOST=0.0.0.0 ollama serve
 ```
 
-Then, you'll need to pull the `llama3.2` model, which is used by the *Manugen AI* package for invoking tools and generating text.
+Make sure the Ollama server is accepting requests by inspecting the logs.
+The command above should output something like `"Listening on [::]:11434"` (it *should not* include `127.0.0.1`, otherwise we won't be able to connect to it from a Docker container).
+To check the logs in OS X, you can run the following command to tail the server logs: `tail -f ~/.ollama/logs/server.log`.
+
+Then, you'll need to pull a model of your choice, which is used by the *Manugen AI* package for invoking tools, generating text, and interpreting figures.
+Make sure the model you pick [supports "tools"](https://ollama.com/search?c=tools), such as [Qwen3](https://ollama.com/library/qwen3):
 
 ```bash
-ollama pull llama3.2
+# qwen3 supports tools and thinking
+ollama pull qwen3:8b
 ```
 
-In OS X, if you'd like to see if your Ollama server is accepting requests, you can run the following command to tail the server logs:
+If you want to use Manugen-AI to upload figures and interpret them, you'll also need a model that [supports "vision"](https://ollama.com/search?c=vision), such as [Gemma3](https://ollama.com/library/gemma3):
 
 ```bash
-tail -f ~/.ollama/logs/server.log
+# gemma3 supports vision, which can be used to interpret your figures
+ollama pull gemma3:4b
 ```
 
+For the sake of keeping computational demands low, **the models suggested here are small and may not yield the best results.**
+You'll need to test which model works best for your task.
+In general, larger models perform better (you can try larger versions of Qwen3 and Gemma3), provided you have a GPU with sufficient VRAM to maintain acceptable inference times.
```

Comment:
Adding in "high context size" here - this seemed to play a part in quality outcomes with ollama models when I tested a few.

```suggestion
In general, we've observed that large models or those with high context size perform better (you can try larger versions of Qwen3 and Gemma3), provided you have a GPU with sufficient VRAM to maintain acceptable inference times.
```